android-components/components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/String.kt

```diff
@@ -405,3 +412,11 @@ fun String.last4Digits(): String {
 fun String.trimmed(): String {
     return this.take(MAX_URI_LENGTH)
 }
+
+fun String.base64ToBitmap(prefix: String = IMAGE_URI_PREFIX): Bitmap? {
```

Comment:
Nit: Let's add some docs to document how this function works, we could add some unit tests too :)